.gitignore

```diff
@@ -132,4 +132,7 @@ dmypy.json
 config/config.yml
 config/config.env
 
-docker-compose.dev.yml
\ No newline at end of file
+docker-compose.dev.yml
+.idea
```

Comment:
Details about your local dev environments shouldn't find place in a project `.gitignore`. 
Try adding these to your machine's global `.gitignore`

Reply:
removed

Reply:
I guess .idea is ok.